app/services/case_creator.rb

```diff
@@ -0,0 +1,65 @@
+require 'glimr_api_client'
+
+class CaseCreator
+  attr_reader :tribunal_case, :errors
+  attr_reader :tc_number, :conf_code, :payment_url
+
+  alias_attribute :tc, :tribunal_case
+
+  def initialize(tribunal_case)
+    @tribunal_case = tribunal_case
+    @errors = []
+  end
+
+  def call
+    create_glimr_case!
```

Comment:
We also need to build the documentsUrl parameter, before we make the call to GLiMR. Could we have some code for that here, even if it's just a TODO comment, for now?